### PR TITLE
Enable --cluster-monitoring flag description

### PR DESCRIPTION
**Pull request by flacatus**
### What does this PR do?
This PR enables cluster monitoring description in Readme.md. Also change default olm suggested namespace name(`openshift-workspaces`).
Upstream related PR: https://github.com/che-incubator/chectl/pull/966

### CODE REVIEW

**Comment by flacatus** [comment]
cc @tolusha

File: build/scripts/sync-chectl-to-crwctl.sh [hunk 1]
```diff
@@ -129,9 +129,9 @@ installerString="    installer: string({\n\
 clusterMonitoringString="    'cluster-monitoring': boolean({\n\
       default: false,\n\
       hidden: false,\n\
-      description: \`Enable cluster monitoring to scrape metrics in Prometheus.\n\
+      description: \`Enable cluster monitoring to scrape CodeReady Workspaces metrics in Prometheus.\n\
```
Comment (nickboldt) [inline build/scripts/sync-chectl-to-crwctl.sh:132]: do we need to deviate from upstream here?

File: build/scripts/sync-chectl-to-crwctl.sh [hunk 2]
```diff
@@ -125,6 +126,13 @@ installerString="    installer: string({\n\
       description: 'Installer type. If not set, default is "olm" for OpenShift >= 4.2, and "operator" for earlier versions.',\n\
       options: ['olm', 'operator']\n\
     }),"; # echo -e "$installerString"
+clusterMonitoringString="    'cluster-monitoring': boolean({\n\
```
Comment (nickboldt) [inline build/scripts/sync-chectl-to-crwctl.sh:129]: for easier maintenance in future, this block could be read from upstream, then sed-replaced:

```
clusterMonitoringString="$(grep -A5 "'cluster-monitoring': boolean" src/commands/server/deploy.ts | sed -r -e "s/hidden: true/hidden: false/g")"; # echo -e "$clusterMonitoringString"
```

**Review by nickboldt — CHANGES_REQUESTED** [review]
see inline comments

File: build/scripts/sync-chectl-to-crwctl.sh [hunk 3]
```diff
@@ -134,11 +142,18 @@ pushd "${TARGETDIR}" >/dev/null
...
+
 		# Remove --domain flag
 		sed -i '/domain: string({/,/}),/d' "${TARGETDIR}/${d}"
+
 		# Change multi-user flag description. Code Ready Workspaces support multi-user by default. https://issues.redhat.com/browse/CRW-1174
 		sed -i "s|'Starts CodeReady Workspaces in multi-user mode'|\`Deploys CodeReady Workspaces in multi-user mode.\n\ \
 		                Note, this option is turned on by default.\`|g" "${TARGETDIR}/${d}"
+
+		# Enable cluster monitoring description in Readme. Cluster Monitoring actually is available only for downstream
+		perl -0777 -p -i -e 's|(\ +'"'cluster-monitoring'"': boolean\({.*?}\),)| ${1} =~ /.+openshift.+/?"INSERT-CONTENT-HERE":${1}|gse' "${TARGETDIR}/${d}"
```
Comment (nickboldt) [inline build/scripts/sync-chectl-to-crwctl.sh:154]: does this change depend on some uncommitted upstream change? I don't see cluster-monitoring in https://github.com/che-incubator/chectl/blob/master/src/commands/server/deploy.ts

Comment (flacatus) [reply build/scripts/sync-chectl-to-crwctl.sh:154]: It is not merged yet. 

che-incubator/chectl#966

File: build/scripts/sync-chectl-to-crwctl.sh (hunk 1 above)
Comment (tolusha) [reply build/scripts/sync-chectl-to-crwctl.sh:132]: Not in this case. We hide this option in chectl. That's why we have to readd in downstream.m

Comment (nickboldt) [reply build/scripts/sync-chectl-to-crwctl.sh:132]: so no need to inject "CodeReady Workspaces" into this line.

File: build/scripts/sync-chectl-to-crwctl.sh (hunk 3 above)
Comment (nickboldt) [reply build/scripts/sync-chectl-to-crwctl.sh:154]: Narrator: it was merged.

File: build/scripts/sync-chectl-to-crwctl.sh (hunk 2 above)
Comment (flacatus) [reply build/scripts/sync-chectl-to-crwctl.sh:129]: @nickboldt How we spoke recently if I'll do in this way I'll have some problems by applying changes with sed but I have a lot of problems with sed.
I've tried for example 
perl -0777 -p -i -e 's|(\ +'"'cluster-monitoring'"': boolean\({.*?}\),)| ${1} =~ /.+openshift.+/?"INSERT-CONTENT-HERE":${1}|gse' "${TARGETDIR}/${d}"
```sed -r -e "s|INSERT-CONTENT-HERE|"\\\\ \\${clusterMonitoringString}$"|g" -i "${TARGETDIR}/${d}"```
Can you help me please to find the right sed command?

**Review by nickboldt — APPROVED** [review]

**Pull request merged**